models/app.go

```diff
@@ -1,6 +1,13 @@
 package models
 
 // App is a serverless application from the repository
+// Stores the json file format of the response
+// Fields in structures can have tags, which hold metadata about a field.
+// The `json:"VALUE"` tag tells the JSON library to place the value of that field under the `VALUE` key in the json object.
```

Comment:
Question, do we want to include this sort of information in our documentation? This is a well known Go pattern which is documented in the core documentation of Go.

Reply:
it was just to make it easier for a non-golang person to read the code

Reply:
sure